src/components/Sologan/Sologan.jsx

```diff
@@ -0,0 +1,110 @@
+import { Box, Stack, Typography } from '@mui/material'
+
+
+export default function Sologan() {
+
+    return (
+        <Box
+            sx={
+                {
+                    height: "23.8125rem",
+                    width: "100%",
+                    backgroundColor: "#F4FFEB",
+                    position: "relative",
+                    zIndex: "0",
+
+                }
+            }
+        >
+            <Box
+                sx={{
+                    position: "absolute",
+                    left: "50%",
+                    bottom: "3.44rem",
+                    transform: "translateX(-50%)",
+                    zIndex: "1",
+                    width: "72.6875rem",
+                    height: "23.8125rem",
+                    borderRadius: "0.625rem",
+                    backgroundColor: "#FFF",
+                    display: "flex",
+                    flexDirection: "column",
+                    gap: "2.44rem",
+                    alignItems: "center",
+                    p: "1.75rem 0 2.31rem 0"
+                }}
+            >
+                <Stack
+                    direction="column"
+                    alignItems="center"
+                >
+                    <Typography variant="body1" sx={{
+                        fontStyle: "italic", color: "#69AD28", fontWeight: "200", fontSize: "1.25rem",
+                        position: "relative",
+                        '&::before, &::after': {
+                            content: '""',
+                            position: 'absolute',
+                            top: "50%",
+                            transform: "translateY(-50%)",
+                            height: '1px',
+                            width: "4rem",
+                            backgroundColor: '#69AD28', // Màu sắc của đường gạch ngang
+                        },
+                        '&::before': {
+                            right: '12.5rem', // Khoảng cách giữa chữ và đường gạch ngang bên trái
+                        },
+                        '&::after': {
+                            left: '12.5rem', // Khoảng cách giữa chữ và đường gạch ngang bên phải
+                        },
+                    }}>
+                        Chào mừng bạn đến
+                    </Typography>
+                    <Typography variant='h5' sx={{ fontSize: "2.5rem", fontWeight: "400", color: "#214400", fontFamily: 'FS PF BeauSans Pro, sans-serif' }}>PLANTMED</Typography>
+                </Stack>
+                <Stack
+                    direction="column"
+                    alignItems="center"
+                    spacing={2}
+                    sx={
+                        {
+                            width: "58.0625rem"
+                        }
+                    }
```

Comment:
You have a width style only here, but you lose 4 lines for it. Let's reduce the code line